shell/app/modules/project/common/components/issue/edit-issue-drawer.tsx

```diff
@@ -128,6 +128,29 @@ const IssueMetaFields = ({ labels, isEditMode, isBacklog, editAuth, issueType, f
     ));
   }, [taskTypeList]);
 
+  React.useEffect(()=>{
+    if(ref && !ref.current){
+      const customFieldKeys = map(customFieldDetail?.property || [], 'propertyName').concat(['taskType', 'bugStage', 'owner']);
```

Comment:
no need `|| []`,    `map(undefined, 'xxx')` output is `[]`

Reply:
to prevention ```customFieldDetail.property = null```